packages/core/src/events/DefaultEventHandlers.ts

```diff
@@ -160,7 +160,18 @@ export class DefaultEventHandlers<O = {}> extends CoreEventHandlers<
             store.actions.setIndicator(indicator);
           }
         );
+        
+        const unbindDragLeave = this.addCraftEventListener(
+          el,
+          'dragleave',
+          (e) => {
+            e.craft.stopPropagation();
+            e.preventDefault();
 
+            this.positioner?.removeIndicator();
```

Comment:
Could you remove optional chaining for now? (We're still on an older version of TS) 

```ts
if ( !this.positioner ) {
  return; 
}

this.positioner.removeIndicator();
```

Do you mind explaining to me a little on how this works? It seems we're removing the indicator when the dragged element leaves a droppable element. Let's say if we're dragging some elements within the ROOT element, then we move our cursor out of the ROOT element, would this remove the indicator as well?